.dep-versions

```diff
@@ -1,5 +1,4 @@
-# Update the version check in catalyst.__init__ when changing the JAX version.
-jax=0.4.14
-mhlo=00be4a6ce2c4d464e07d10eae51918a86f8df7b4
-llvm=4706251a3186c34da0ee8fd894f7e6b095da8fdc
-enzyme=8d22ed1b8c424a061ed9d6d0baf0cc0d2d6842e2
+jax=0.4.23
```

Comment:
We should probably keep this comment for the future:
```suggestion
# Always update the version check in catalyst.__init__ when changing the JAX version.
jax=0.4.23
```

Reply:
Good catch!